scripts/processing/06-make_epochs.py

```diff
@@ -112,15 +108,35 @@ def run_epochs(subject_id, tsss=False):
         ica_name = op.join(meg_dir, subject, 'run_concat-ica.fif')
     print('  Using ICA')
     ica = read_ica(ica_name)
+    ica.exclude = []
+
     n_max_ecg = 3  # use max 3 components
-    ecg_epochs = create_ecg_epochs(raw, tmin=-.5, tmax=.5)
+    ecg_epochs = create_ecg_epochs(raw, tmin=-.1, tmax=.1)
+    ecg_epochs.apply_baseline((None, None))
     ecg_epochs.decimate(11, verbose='error')
     ecg_inds, scores_ecg = ica.find_bads_ecg(ecg_epochs, method='ctps',
                                              threshold=0.8)
-    ica.exclude = ecg_inds[:n_max_ecg]
-    ica.save(ica_name)
+    print('    Found %d ECG indices' % (len(ecg_inds),))
+    ica.exclude.extend(ecg_inds[:n_max_ecg])
+
+    n_max_eog = 3  # use max 3 components
+    eog_epochs = create_eog_epochs(raw, tmin=-.1, tmax=.1)
+    eog_epochs.apply_baseline((None, None))
+    eog_epochs.decimate(11, verbose='error')
+    eog_inds, scores_eog = ica.find_bads_eog(eog_epochs)
+    print('    Found %d EOG indices' % (len(eog_inds),))
+    ica.exclude.extend(eog_inds[:n_max_eog])
```

Comment:
why did you do this? Now, we have eyeblinks in the signals

Reply:
Shouldn't this remove any residual blinks?

Reply:
But now you don't have EOG channel in autoreject and also the ICA components are not selected properly if you look at the website

Reply:
Oh you're taking about the lines below, not this one.

In practice it didn't change much IIRC to include EOG in the list (as large EOG creates large EEG, too) but I'll put it back to be safe.

As for component selection I'm not sure how to improve it, ideas welcome. But we should do something as subjects do tend to blink after 800 ms.